salt/netapi/rest_cherrypy/app.py

```diff
@@ -1239,7 +1239,7 @@ def GET(self):
             HTTP/1.1 200 OK
             Content-Type: application/json
         '''
-        import inspect
+        import inspect  # pylint: disable=unused-import
```

Comment:
Actually remove the import?
Or at least document why the import is done since it's not used anywhere in the module?

Reply:
I wasn't sure if it was needed somewhere else, so I was being cautious. Sometimes we do late imports because they are implicitly needed by other code in a given function.

Reply:
yeah, but in those cases we should add a comment. Anyway, not a blocker, leave it, it's safer this way.